jenkins/model_output_processing_Jenkinsfile

```diff
@@ -37,6 +37,10 @@ pipeline {
           wget https://prod-owi-resources.s3-us-west-2.amazonaws.com/resources/Application/wbeep/model_output/${DATE}_hru_intcpstor_out.nc
           wget https://prod-owi-resources.s3-us-west-2.amazonaws.com/resources/Application/wbeep/model_output/${DATE}_pkwater_equiv_out.nc
           wget https://prod-owi-resources.s3-us-west-2.amazonaws.com/resources/Application/wbeep/model_output/${DATE}_soil_moist_tot_out.nc
+		  pattern="*.nc"
```

Comment:
tabbing is wonky

With bash, you typicaly want to do UPPER_CASE_AND_UNDERSCORE